packages/node/src/lib/novu.ts

```diff
@@ -1,10 +1,12 @@
 import { EventEmitter } from 'events';
 import axios, { AxiosInstance } from 'axios';
 import { ITriggerPayload } from './template/template.interface';
+import { Subscribers } from './subscribers/subscribers';
 
 export class Novu extends EventEmitter {
   private readonly apiKey?: string;
   private readonly http: AxiosInstance;
+  subscribers: Subscribers;
```

Comment:
```suggestion
  readonly subscribers: Subscribers;
```

A small one just to denotate this shouldn't be changed or rewritten.